ledger-service/http-json/src/it/edition/ce/com/daml/http/NonRepudiationTest.scala

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.http
+
+import java.util.UUID
+
+import akka.http.scaladsl.model.StatusCodes
+import com.daml.nonrepudiation.CommandIdString
+
+@SuppressWarnings(Array("org.wartremover.warts.NonUnitStatements"))
+final class NonRepudiationTest extends AbstractNonRepudiationTest {
+
+  import HttpServiceTestFixture._
+
+  "fail " in withSetup { (db, uri, encoder) =>
```

Comment:
Just noticed the message here is incomplete. Working on it.

Reply:
Addressed in https://github.com/digital-asset/daml/pull/9219/commits/703142d146a372895fe560f757f1d8f697edc0f5